roles/gpu_operator_bundle_from_commit/files/operator_image_builder_script.yml

```diff
@@ -22,22 +20,37 @@ data:
 
     git show --quiet
 
-    if [ "${BUILDER_FROM_IMAGE}" ]; then
-      # if provided, use custom 'FROM ... as builder' image
-      sed "s|FROM golang:.* as builder|FROM ${BUILDER_FROM_IMAGE} as builder|" -i ${CONTAINER_FILE}
+    CONTAINER_FILE=./Dockerfile
+    if [ -f ${CONTAINER_FILE} ]; then
+      # TODO: Remove this backwards compatible code once we stop testing versions that do not use the new Makefile
+      # This was merged in https://github.com/NVIDIA/gpu-operator/commit/b0e99d9e6ac8738ee335f6e866242b1c9977ea09
+
+      cat <<EOF > /usr/local/bin/docker
+      #! /bin/sh
+      exec podman --cgroup-manager=cgroupfs "\$@"
+      EOF
```

Comment:
EOF should be indented 2 spaces back (align with the 0 indentation of the shell script)

Reply:
OK. Thanks.